jobs/tests/test_utils.py

```diff
@@ -1193,3 +1194,71 @@ async def test_execute_external_pipeline(sign_s3_links: bool):
             )
         else:
             assert FakePipeline.calls[-1]["files"][0].get("signed_url") is None
+
+
+@pytest.mark.asyncio
```

Comment:
Remove this decorator and change this pytest config to `auto`
https://github.com/epam/badgerdoc/blob/2177c64d6ab8b1c8b34228ce207f1f64b9bde702/jobs/pytest.ini#L2
IDK why it was set to `strict` there (default behaviour)